static/js/reducers/bootcamps.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import { GET, INITIAL_STATE } from "redux-hammock/constants"
+import type { Bootcamp } from "../flow/discussionTypes"
+import * as bootcampAPI from "../lib/api/bootcamps"
+
+const updateBootcampHandler = (
+  payload: Bootcamp,
+  data: Map<number, Bootcamp>
+): Map<number, Bootcamp> => {
+  const update = new Map(data)
+  update.set(payload.id, payload)
+  return update
+}
+
+export const bootcampsEndpoint = {
+  name:              "bootcamps",
+  verbs:             [GET],
+  initialState:      { ...INITIAL_STATE, data: new Map() },
+  getFunc:           (id: number) => bootcampAPI.getBootcamp(id),
+  getSuccessHandler: updateBootcampHandler
+}
```

Comment:
I think it's not necessary for this PR, but if you want to use redux-query for managing this API instead of redux-hammock I think that would be a good idea, we're generally migrating away from redux-hammock and towards redux-query. I've been using it for any new APIs in Open for a little bit.

Reply:
If you don't want to take the time now though that's alright, this is a pretty simple API so we can easily change it out later.

Reply:
I think I'd rather work on that for the next PR when I introduce the API's for learning paths and programs.  Will probably need to add `reselect` for redux-query to package.json & yarn.lock, plus rewrite the course API code, and this PR is already pretty big.

Reply:
sounds good, I'm adding reselect and fiddling a bunch with the courses API currently anyhow in my work for #2091